src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see
+ * an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString
+    .split('\n')
+    .filter(string => string.length > 5)
+    .map((el) => {
+      return el.split(':')
+        .map(part => part.trim())
+        .reduce((key, values) => {
+          const obj = {};
+
+          obj[key] = values.slice(0, -1).trim();
```

Comment:
you could split by `;` in the very beginning and avoid this `slice` here

Reply:
and why do you need these `obj`s? why don't use `forEach` on this step and write `key` and `value `directly in the `styles`

Reply:
Thanks for advice. About 'obj':  it was my way of making things harder), but honestly I haven`t tried to put them directly in "styles". I have just fixed my code. Please review it.